src/interfaces/vrep/DQ_VrepInterface_py.cpp

```diff
@@ -73,6 +77,9 @@ void init_DQ_VrepInterface_py(py::module& m)
     //void trigger_next_simulation_step();
     dqvrepinterface_py.def("trigger_next_simulation_step", &DQ_VrepInterface::trigger_next_simulation_step, "Sends a synchronization trigger signal to the server.");
 
+    //void wait_for_simulation_step_to_end(); 
+    dqvrepinterface_py.def("wait_for_simulation_step_to_end", &DQ_VrepInterface::wait_for_simulation_step_to_end, "Returns the time needed for a command to be sent to the server, executed, and sent back.");
```

Comment:
@juanjqo Can we update this comment to say that it `waits` for something? After all, that's the expected usage.

I think this is a replication of the comment in C++, which is also wrong. I'm pretty sure it was me who wrote it wrong to start with, but anyway it's time to fix it.

Reply:
@mmmarinho sure! I updated the comment.